src/symbols/Edge.tsx

```diff
@@ -68,7 +68,7 @@ export const Edge: FC<EdgeProps> = ({
   const { isSelected, hasSelections, hasSingleSelection } = useStore(state => ({
     hasSingleSelection: state.selections?.length === 1,
     hasSelections: state.selections?.length,
-    isSelected: state.selections?.includes(fromId)
+    isSelected: state.selections?.includes(fromId) || state.selections?.includes(toId) || state.selections?.includes(id)
```

Comment:
We don't want to test the `toId` because that creates oddness with the arrow directions. The selections change is good though.

```suggestion
    isSelected: state.selections?.includes(fromId) || state.selections?.includes(id)
```

Reply:
Okay, thanks for the review